satpy/avhrr_l1b_gaclac.ipynb

```diff
@@ -0,0 +1,261 @@
+{
```

Comment:
<p>The link to pygac github actually points to a specific PR, is that then way you want it ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pytroll/pytroll-examples/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Yes. A couple of people were asking me about reading LAC data with pygac. But since there hasn't been much progress on the LAC reader (no offence :)) I wanted to encourage users to contribute.</p>